probe_scraper/check_repositories.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+import requests as reqs
+import yaml
+
+GITHUB_RAW_URL = "https://raw.githubusercontent.com"
+REPOSITORIES = os.path.join(
+    os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "repositories.yaml"
+)
+
+with open(REPOSITORIES) as data:
+    repos = yaml.load(data, Loader=yaml.FullLoader)
+    for repo in repos:
+        repo_url = repos[repo]["url"]
+        branch = "master"
+        if "branch" in repos[repo]:
+            branch = repos[repo]["branch"]
+        metrics_files = []
+        if "metrics_files" in repos[repo]:
+            metrics_files = repos[repo]["metrics_files"]
+        if metrics_files:
+            temp_url = (
```

Comment:
This would probably be a bit clearer with an f-string: https://realpython.com/python-f-strings/